crates/package.json

```diff
@@ -0,0 +1,3 @@
+{
+	"name": "crates"
+}
```

Comment:
Can we add a comment here explaining why we have a package.json in `crates`?

Reply:
Good idea, added :+1: